core/checks.py

```diff
@@ -1,158 +1,122 @@
-        database.Administrators.TierLevel >= 4
-    )
+    query = database.Administrators.select().where(database.Administrators.TierLevel >= 4)
     for admin in query:
         adminIDs.append(admin.discordID)
 
     return ctx.author.id in adminIDs
 
-
 is_botAdmin4 = commands.check(predicate_LV4)
 
 
 def slash_is_bot_admin():
     def predicate(interaction: discord.Interaction) -> bool:
         admin_ids = []
 
-        query = database.Administrators.select().where(
-            database.Administrators.TierLevel >= 1
-        )
+        query = database.Administrators.select().where(database.Administrators.TierLevel >= 1)
         for admin in query:
             admin_ids.append(admin.discordID)
 
         return interaction.user.id in admin_ids
 
-    return discord.app_commands.check(predicate)
+    return app_commands.check(predicate)
 
 
 def slash_is_bot_admin_2():
     def predicate(interaction: discord.Interaction) -> bool:
         admin_ids = []
 
-        query = database.Administrators.select().where(
-            database.Administrators.TierLevel >= 2
-        )
+        query = database.Administrators.select().where(database.Administrators.TierLevel >= 2)
         for admin in query:
             admin_ids.append(admin.discordID)
 
         return interaction.user.id in admin_ids
 
-    return discord.app_commands.check(predicate)
+    return app_commands.check(predicate)
 
 
 def slash_is_bot_admin_3():
     def predicate(interaction: discord.Interaction) -> bool:
         admin_ids = []
 
-        query = database.Administrators.select().where(
-            database.Administrators.TierLevel >= 3
-        )
+        query = database.Administrators.select().where(database.Administrators.TierLevel >= 3)
         for admin in query:
             admin_ids.append(admin.discordID)
 
         return interaction.user.id in admin_ids
 
-    return discord.app_commands.check(predicate)
+    return app_commands.check(predicate)
 
 
 def slash_is_bot_admin_4():
     def predicate(interaction: discord.Interaction) -> bool:
         admin_ids = []
 
-        query = database.Administrators.select().where(
-            database.Administrators.TierLevel >= 4
-        )
+        query = database.Administrators.select().where(database.Administrators.TierLevel >= 4)
         for admin in query:
             admin_ids.append(admin.discordID)
 
         return interaction.user.id in admin_ids
 
-    return discord.app_commands.check(predicate)
-
-
-def mktCommissionAdd(ctx):
-    rolesID = [
-        MKT_ID.r_designManager,
-        MKT_ID.r_designTeam,
-        MKT_ID.r_discordManager,
-        MKT_ID.r_discordTeam,
-        MKT_ID.r_contentCreatorManager,
-    ]
-
-    return any(role.id in rolesID for role in ctx.author.roles)
-
-
-is_mktCommissionAuthorized = commands.check(mktCommissionAdd)
+    return app_commands.check(predicate)
 
 
-def TimmyBetaHost(ctx):
+def timmy_beta_host(ctx):
```

Comment:
Requires inner predicate function?

Reply:
I think puncher may know (im not sure :/)

Reply:
Hm alright, I thought it was a normal check at first but then it was placed in the `commands.check` function so I'm not sure if that's intentional or not. I'll check the documentation real quick.

Reply:
Never mind it's fine